packages-ts/starknet-gauntlet-starkgate/src/commands/L1-bridge/deploy.ts

```diff
@@ -0,0 +1,33 @@
+import { EVMExecuteCommandConfig, EVMExecutionContext, makeEVMExecuteCommand } from '@chainlink/evm-gauntlet'
+import { CATEGORIES } from '../../lib/categories'
+import { l1BridgeContractLoader, CONTRACT_LIST } from '../../lib/contracts'
+
+type UserInput = {}
+type ContractInput = []
+
+const makeUserInput = async (flags, args): Promise<UserInput> => {
+  if (flags.input) return flags.input as UserInput
+
+  return {}
+}
+
+const makeContractInput = async (input: UserInput, context: EVMExecutionContext): Promise<ContractInput> => {
+  return []
+}
+
+const commandConfig: EVMExecuteCommandConfig<UserInput, ContractInput> = {
+  contractId: CONTRACT_LIST.L1_BRIDGE,
+  category: CATEGORIES.L1_BRIDGE,
+  action: 'deploy',
+  ux: {
+    description: 'Deploys an L1 token bridge',
+    examples: [`${CATEGORIES.L1_BRIDGE}:deploy --network=<NETWORK>`],
+  },
+  makeUserInput,
+  makeContractInput,
+  validations: [],
+  loadContract: l1BridgeContractLoader,
+}
+
+// todo: fix type annotation required by compiler
+export default makeEVMExecuteCommand(commandConfig) as any
```

Comment:
There's something to fix inside the EVM gauntlet pkg with the `<CI>` generic, for now let's do:
```suggestion
type ContractInput = any[]
...
export default makeEVMExecuteCommand(commandConfig)
```